CONTRIBUTING.md

```diff
@@ -26,3 +26,7 @@ information on using pull requests.
 
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google/conduct/).
+
+## Development
+
+See the [Development Guidelines](DEVELOPMENT.md).
```

Comment:
This could be added to the `README.md` instead.

Alternatively, you could link to the `CONTRIBUTING.md` from there.

Reply:
link the CONTRIBUTING.md in the README.md